mysql-dao/src/main/resourses/application.properties

```diff
@@ -0,0 +1,7 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/testdb
```

Comment:
Already you added this properties in contentservice properties then why should we need this ?

Reply:
Removed "spring.datasource.url=jdbc:mysql://localhost:3306/testdb" from application.properties file inside mysql-dao module.